connectors/automation-anywhere/src/main/java/io/camunda/connector/automationanywhere/AutomationAnywhereConnector.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.automationanywhere;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.api.client.http.HttpRequestFactory;
+import io.camunda.connector.api.annotation.OutboundConnector;
+import io.camunda.connector.api.json.ConnectorsObjectMapperSupplier;
+import io.camunda.connector.api.outbound.OutboundConnectorContext;
+import io.camunda.connector.api.outbound.OutboundConnectorFunction;
+import io.camunda.connector.automationanywhere.model.AutomationAnywhereRequest;
+import io.camunda.connector.http.base.components.HttpTransportComponentSupplier;
+import io.camunda.connector.http.base.services.HttpService;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Optional;
+
+@OutboundConnector(
+    name = "Automation Anywhere Outbound Connector",
+    inputVariables = {
+      "authenticationStrategy",
+      "url",
+      "method",
+      "authentication",
+      "headers",
+      "queryParameters",
+      "connectionTimeoutInSeconds",
+      "body"
+    },
+    type = "io.camunda:connector-automationanywhere:1")
+public class AutomationAnywhereConnector implements OutboundConnectorFunction {
+  private static final String AUTHENTICATION_HEADER = "X-Authorization";
```

Comment:
Consider using `protected` in order to re-use constant in tests.

```suggestion
  protected static final String AUTHENTICATION_HEADER = "X-Authorization";
```